lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -52,27 +53,36 @@ class LoadFastEnough4Pwa extends Audit {
   static audit(artifacts) {
     const devtoolsLogs = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
     return artifacts.requestNetworkRecords(devtoolsLogs).then(networkRecords => {
-      const allRequestLatencies = networkRecords.map(record => {
-        // Ignore requests that don't have timing data or resources that have
-        // previously been requested and are coming from the cache.
-        // Also ignore unfinished requests since they won't have timing information.
+      const firstRequestLatenciesByOrigin = new Map();
+      networkRecords.forEach(record => {
+        // Ignore requests that don't have valid origin, timing data, came from the cache, or are
+        // not finished.
         const fromCache = record._fromDiskCache || record._fromMemoryCache;
-        if (!record._timing || fromCache || !record.finished) {
-          return undefined;
+        const origin = URL.getOrigin(record._url);
+        if (!origin || !record._timing || fromCache || !record.finished) {
+          return;
         }
 
         // Use DevTools' definition of Waiting latency: https://github.com/ChromeDevTools/devtools-frontend/blob/66595b8a73a9c873ea7714205b828866630e9e82/front_end/network/RequestTimingView.js#L164
         const latency = record._timing.receiveHeadersEnd - record._timing.sendEnd;
-
-        return {
+        const latencyInfo = {
           url: record._url,
+          startTime: record._startTime,
+          origin,
           latency: latency.toLocaleString(undefined, {maximumFractionDigits: 2})
         };
+
+        // Only examine the first request per origin to reduce noisiness from cases like H2 push
+        // where individual request latency may not apply.
+        const existing = firstRequestLatenciesByOrigin.get(origin);
+        if (!existing || existing.startTime > latencyInfo.startTime) {
+          firstRequestLatenciesByOrigin.set(origin, latencyInfo);
+        }
       });
 
+      const firstRequestLatencies = Array.from(firstRequestLatenciesByOrigin.values());
       const latency3gMin = Emulation.settings.TYPICAL_MOBILE_THROTTLING_METRICS.targetLatency - 10;
-      const areLatenciesAll3G = allRequestLatencies.every(val =>
-          val === undefined || val.latency > latency3gMin);
+      const areLatenciesAll3G = firstRequestLatencies.every(val => val.latency > latency3gMin);
```

Comment:
i missed this before but we're doing a `>` comparison on strings.

let's keep the `latency` value from above, separately from the toLocaleString() version..